src/pspm_sf.m

```diff
@@ -9,7 +9,7 @@
 %   ├───────.datafile:  one data filename or cell array of filenames.
 %   ├──────.modelfile:  one data filename or cell array of filenames.
 %   ├─────────.timing:  can be one of the following
-%   │                   - an SPM style onset file with two event types: onset &
+%   │                   - an SPM style onset file with two events types: onset &
```

Comment:
This is because I noticed the marker channel is always selected through the first channel (https://github.com/bachlab/PsPM/pull/493/files#r1237991145). In this case it is *a* marker event channel. I understand this as an "event" so I renamed the original `marker_channel_events` to be `marker_channel_event`. However I noticed some test data may expect this to be multiple channels so I revert `event` back to `events`. This sentence is an incident that happens after I made full file replacement where I did not notice this line. I apologise for this mistake.